STYLE-GUIDE.md

```diff
@@ -2,22 +2,41 @@
 
 This document acts as a Style Guide for the language and wording used in exercises.
 
-## General principle
+## Application
 
-All content should be written in US English.
+These rules should be followed in all exercises. 
+Existing descriptions and test-cases can be updated to adhere to them without requiring "replacement" test cases.
+
+### Consistency within an exercise.
+
+There are some terms that have multiple valid spellings (e.g. "lower case" vs "lowercase"). 
+Where a consistent style has not been agreed within this document, these must be consistent within an exercise.
+
+### Exceptions
+
+Exercises may vary from these rules if there is general consensus across maintainers that this is reasonable. 
+For example, an exercise about converting between different units might choose not to use SI measurements.
+
+## Language
+
+All content should be written in US English. In the future other translations may occur, but the "official" Exercism language is US English.
```

Comment:
I was wondering if there is a resource we could link to that lists some of the differences between US English and UK English? That way people have a convenient resource to check their text against.

Reply:
Yeah - can you put a seperate issue together for someone to do this pls?